site/ip-addresses.md

```diff
@@ -0,0 +1,88 @@
+---
+title: IP Addresses, routes and networks
+layout: default
+---
+
+# IP Addresses, routes and networks
+
+Weave Net lets you run containers on a private network, and so the IP
+addresses those containers use are insulated from the rest of the
+Internet, and you don't have to worry about them clashing. Except, if
+they actually do clash with some addresses that you'd like the
+containers to talk to.
+
+Some definitions:
+
+- _IP_ is the Internet Protocol, the fundamental basis of network
+   communication between billions of connected devices.
+- The _IP address_ is (for most purposes) the four numbers separated
+  by dots, like `192.168.48.12`. Each number is one byte in size, so can
+  be between 0 and 255.
+- Each IP address lives on a _Network_, which is some set of those
+  addresses that all know how talk to each other. The network address
+  is some prefix of the IP address, like `192.168.48`. To indicate
+  which part of the address indicates the network, we append a slash
+  and then the number of bits in the network prefix, like
+  `/24`. Another example would be `10.4.2.6/8`, where the network
+  prefix is the first 8 bits - `10`. Written out in full, the address
+  of this network is `10.0.0.0/8`. The most common prefix lengths are
+  8, 16 and 24, but there is nothing stopping you using a /9 network
+  or a /26.
```

Comment:
You may want to give an example here, rather than below. I reckon it's those non-byte-aligned network prefixes that really confuse people.

We may also want to link to one of those CIDR calculators, such as http://www.ipaddressguide.com/cidr (that one can even be embedded).